demo/api/src/db/fixtures/generators/blocks/media/media-gallery-list-block-fixture.service.ts

```diff
@@ -0,0 +1,28 @@
+import { ExtractBlockInputFactoryProps } from "@comet/blocks-api";
+import { faker } from "@faker-js/faker";
+import { Injectable } from "@nestjs/common";
+import { MediaGalleryListBlock } from "@src/common/blocks/media-gallery.block";
+
+import { MediaGalleryItemBlockFixtureService } from "./media-gallery-item-block-fixture.service";
+
+@Injectable()
+export class MediaGalleryListBlockFixtureService {
```

Comment:
Another benefit of grouping services is that we don't need to export internals like MediaGalleryListBlock.